stl/inc/valarray

```diff
@@ -195,99 +184,175 @@ public:
     valarray& operator=(const indirect_array<_Ty>& _Indarr); // defined below
 
     _NODISCARD valarray operator+() const {
-        _VALOP(_Ty, size(), +_Myptr[_Idx]);
+        valarray<_Ty> _Ans(size());

```

Comment:
Should we capture `size()` so that we do not need the call to `_Ans.size((`

Reply:
I think as `_Ans` is a local variable, compiler can optimize `size()` to not reading from memory address by itself

Reply:
But debug  performance. 

I am totally fine with it, given that it is existing code

Reply:
Right, debug will be affected...

Reply:
Done.
(If maintainers disagree, I'll revert)